lib/pages/chat/chat.dart

```diff
@@ -175,7 +178,7 @@ class ChatController extends State<Chat> with ImagePickerMixin, SendFilesMixin {
     );
     final roomId = success.result;
     if (roomId == null) return;
-    VRouter.of(context).toSegments(['rooms', roomId]);
+    context.go('/rooms/$roomId');
```

Comment:
please define the constants of route components
then we can easy to update in future
```
context.goRoom(roomId)
void goRoom(String roomId) {
   go('/$roomPath/$roomId')
}
```
But it is just suggestion, can do it later in a refactor PR

Reply:
IMO:

For the chat list screen, we only need to pay attention to the other screens as follows.

/room/abc/
/room/xyz/

We also have support for the web, so this section can be updated for the better. But now I don't want to fix it because this PR is too big